GCMtools/utils/read_and_write.py

```diff
@@ -42,7 +42,10 @@ def m_read_raw(gcmt, gcm, data_path, iters='last', load_existing=False, tag=None
         from GCMtools.exorad import m_read_from_mitgcm
 
         if tag is not None and load_existing:
-            loaded_ds = gcmt.get_models(tag)
+            try:
+                loaded_ds = gcmt.get_models(tag)
+            except KeyError:
+                loaded_ds = None
```

Comment:
This should throw at least a warning that the tag was not loaded

Reply:
Thats already happening at a different place in load. I think it should be fine that way.